### PR TITLE
Added hackathon content to product summary

### PR DESCRIPTION
**Pull request by janetvong**

### CODE REVIEW

File: TechnicalProductSummary.rst [hunk 1]
```diff
@@ -19,3 +19,24 @@ Starting notes:
 
 Hack-a-thon content:
 --------------------
+When you're working on creating a product summary for a technical product, your first step is to brainstorm your plan. 
+
+Brainstorm
+You should prepare an outline that answer's the the 5 W's.
```
Comment (jlara310) [inline TechnicalProductSummary.rst]: answer's -> answers

File: TechnicalProductSummary.rst [hunk 2]
```diff
@@ -19,3 +19,24 @@ Starting notes:
...
+* What should the summary cover? 
+* When is the writing deadline? What does the roadmap timeline look like in order to help you meet the writing deadline? 
+* Why is the summary needed? 
+* Where is the summary going to live? How will your audience find the content? 
+
+By asking these question, you create the scope for your writing project. 
+
+Next, outline what information should be captured in your summary. If there is exisiting documentation, such as design docs, guides, etc. read or glimpse through those documents to get a sense of the product. You can also use Google search to gain background and introductory information on the subject area. List key words and key concepts the reader might need to know before they start reading the summary. 
+
+Then, think about the questions to ask your subject matter expert (SME). Plan an outline to strategize your structure of your summary, and phrase your interview questions to help you fill content into the structure of your doc.
```
Comment (jlara310) [inline TechnicalProductSummary.rst]: Suggestion: "strategize your structure of your summary" -> "stragetize the structure of your summary"

**Comment by jlara310** [comment]
Thanks!

File: TechnicalProductSummary.rst [hunk 3]
```diff
@@ -19,3 +19,24 @@ Starting notes:
 
 Hack-a-thon content:
 --------------------
+When you're working on creating a product summary for a technical product, your first step is to brainstorm your plan.
```
Comment (scotsmatrix) [inline TechnicalProductSummary.rst]: This needs an introduction, as suggested by the "Starting notes".  For example:

Check with Marketing and Sales to see if a technical product summary already exists. If there isn't a summary, you should create one.

File: TechnicalProductSummary.rst [hunk 4]
```diff
@@ -19,3 +19,24 @@ Starting notes:
...
+
+Then, think about the questions to ask your subject matter expert (SME). Plan an outline to stragetize the structure of your summary, and phrase your interview questions to help you fill content into the structure of your doc. 
+
+Now that you have done your research and have interviewed your subject matter expert, you should have enough information to get started with writing the summary. Start by introducing the topic. Think about the purpose of the product and the value it brings to its users. Evangelize the product. If the technical product summary is intended for a technical audience, describe the basic concepts or prerequisites that the reader might need to know before they jump into writing. Use your outline to structure your summary. Feel free to mention similar products, the competitors, or what sets this product apart from other products. Use visuals if it helps the reader understand the product. 
+
```
Comment (scotsmatrix) [inline TechnicalProductSummary.rst]: I'm not sure I agree with "Evangelize the product."  A product technical summary is not a marketing document. The description can be "marketing-friendly" but remember the target audience is the technical readership in a prospect or customer organization.

**Pull request merged**